src/renderer/views/SubscribedChannels/SubscribedChannels.css

```diff
@@ -50,7 +50,6 @@
   text-overflow: ellipsis;
   overflow-wrap: break-word;
   inline-size: 100%;
-  display: box;
```

Comment:
This should actually changed back to `display: -webkit-box;`
![image](https://github.com/user-attachments/assets/41576445-56d7-4d98-b7af-174a857028b6)

With `box` or removed
![image](https://github.com/user-attachments/assets/ba21357d-09fa-4d88-8cf8-3c79789374be)